projects/netgrif-components/src/lib/data-fields/filter-field/filter-string-query-field/filter-string-query-field.component.html

```
@@ -0,0 +1,9 @@
+<mat-form-field [appearance]="dataField.materialAppearance" class="full-width" color="primary">
+    <mat-label *ngIf="!showLargeLayout.value">{{dataField.title}}</mat-label>
+    <input matInput
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)